qa-brh.planx-pla.net/manifest.json

```diff
@@ -15,16 +15,16 @@
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
     "hatchery": "quay.io/cdis/hatchery:1.0.0",
     "indexd": "quay.io/cdis/indexd:2022.06",
-    "kayako-wrapper": "quay.io/cdis/kayako-wrapper-service:master",
+    "kayako-wrapper": "quay.io/cdis/kayako-wrapper-service:0.1.1",
     "manifestservice": "quay.io/cdis/manifestservice:2022.06",
     "metadata": "quay.io/cdis/metadata-service:BRH-1.0.0",
     "peregrine": "quay.io/cdis/peregrine:2022.06",
     "pidgin": "quay.io/cdis/pidgin:2022.06",
-    "portal": "quay.io/cdis/data-portal:feature_BRH-259_fix_wts_portal_logic",
+    "portal": "quay.io/cdis/data-portal:5.0.0",
     "requestor": "quay.io/cdis/requestor:1.7.1",
     "revproxy": "quay.io/cdis/nginx:2022.06",
     "sheepdog": "quay.io/cdis/sheepdog:2022.06",
-    "wts": "quay.io/cdis/workspace-token-service:0.4.0"
+    "wts": "quay.io/cdis/workspace-token-service:0.4.1"
```

Comment:
LGTM. The build is currently failing for 0.4.1 (https://github.com/uc-cdis/workspace-token-service/pull/47). As soon as that's working we can merge this in

Reply:
This new 0.4.1 has wts deployment errors in the pod per Jenkins tests, some `ImagePullBackOff` error. Can you look at the logs in qa? Happy to look at it together, we can post in pe-on-call ask for some more guidance.